fferr/misc.go

```diff
@@ -67,3 +76,20 @@ func NewParsingError(err error) *ParsingError {
 type ParsingError struct {
 	baseGRPCError
 }
+
+type UnimplementedError struct {
+	baseGRPCError
+}
+
+func NewUnimplementedErrorf(format string, a ...any) *UnimplementedError {
+	err := fmt.Errorf(format, a)
```

Comment:
same: `(format, a...)`

Reply:
this. Goland doesn't show it but this ones a point for vscode